ios/tools/prepRelease.sh

```diff
@@ -89,9 +89,15 @@ cd $WORK_DIR
 KEYMANAPP_IPA="build/Build/Products/Release-iphoneos/Keyman.ipa"
 KEYMANAPP_IPA_DST="keyman-ios-${BUILD_NUMBER}.ipa"
 
-echo "Copying Keyman IPA ${KEYMANAPP_IPA} => ${UPLOAD_DIR}/${KEYMANAPP_IPA/DST}..."
+echo "Copying Keyman IPA ${KEYMANAPP_IPA} => ${UPLOAD_DIR}/${KEYMANAPP_IPA_DST}..."
 cp "${KEYMANAPP_IPA}" "${WORK_DIR}/${UPLOAD_DIR}/${KEYMANAPP_IPA_DST}"
 
+KEYMANAPP_SIM_IPA="build/Build/Products/Release-iphonesimulator/Keyman.ipa"
```

Comment:
This and the `FIRSTVOICESAPP_SIM_IPA` variable indicate exactly where the new artifacts will be when the main build completes.  It's a matter of preserving after the build.

Reply:
Yep, just set "General Settings:" --> "Artifact paths" on the appropriate build config

Reply:
I've updated the `Test-14.0` build configuration to save all artifacts in the upload/ folder (this was already the case for the `Release-14.0` build configuration. That should cover all of these files. (There's a question in my mind if we should both preserving the 'real device' .ipa given it's effectively useless as an artifact... but that can continue as a separate discussion)

Reply:
See #4248 - the .ipa files are useful for symbolifying crash reports.